.circleci/verbatim-sources/binary-build-params.yml

```diff
@@ -15,6 +15,7 @@ binary_linux_build_params: &binary_linux_build_params
   environment:
     BUILD_ENVIRONMENT: << parameters.build_environment >>
     LIBTORCH_VARIANT: << parameters.libtorch_variant >>
+    ANACONDA_USER: pytorch
```

Comment:
Hmm?

Reply:
default user is soumith which is the conda channel name, but channel soumith is missing dependency to build cuda 10.1. @soumith pointed out that correct user we should use is "pytorch"